### DISCUSSION TITLE
Error when trying to generate indexed report

### DISCUSSION THREAD

**Discussion by flyingml** [sepinf-inc/IPED#2097] [General]
Hello,

I'm getting an error message when I try to generate an indexed report. I believe it has something to do with the audio transcription feature.

Any ideas to solve this?

log: [IPED-2024-02-19-10-07-09.log](https://github.com/sepinf-inc/IPED/files/14378651/IPED-2024-02-19-10-07-09.log)
![MicrosoftTeams-image](https://github.com/sepinf-inc/IPED/assets/1653433/a5540638-253c-4e8e-903b-76cf3b4ede48)

**Comment by lfcnassif** [comment]
The error happened simply because you pointed the output to an existing folder:
`Processing Error: Directory already exists: D:\Análises\3 - ISAAC (I-Phone cinza)\Contatos\iped`

Furthermore, to generate a report, you should create and select at least 1 bookmark.

**Reply by flyingml** [reply]
Hello, 

Sorry I attached the wrong log file.

here it is:  [IPED-2024-02-22-16-31-34.log](https://github.com/sepinf-inc/IPED/files/14384790/IPED-2024-02-22-16-31-34.log)

**Reply by lfcnassif** [reply]
Indeed, it is related to the audio transcription module using Microsoft Azure service. You have 3 options:
1. Go to case/iped/IPEDConfig.txt and set `enableAudioTranscription = false`
2. Create a plugin folder inside the case folder and copy https://csspeechstorage.blob.core.windows.net/maven/com/microsoft/cognitiveservices/speech/client-sdk/1.19.0/client-sdk-1.19.0.jar there
3. If you want to run a different transcription provider, follow the steps inside case/iped/conf/AudioTranscriptConfig.txt

**Reply by flyingml** [reply]
|It worked. Thanks!
